### ISSUE TITLE
AUR version should not have precedence over repo pkgs

### ISSUE THREAD

**Issue by oberon2007** [manjaro/pamac#353]
Yesterday I created new packages **sndio** and **sidplay2-libs** which both exist in the AUR.
Since they are new in the repo, I am using pkgrel 1 although AUR versions have higher pkgrel.
Now while all is well with **pacman** and **yaourt**, **pamac** turns the try icon to red and shows these packages under 'updates'. While it is impossible for us to keep track of AUR versions of all our packages, people shouldn't be encouraged to install AUR versions over repo versions anyway.
```
$ sudo pacman -Syu
:: Synchronising package databases...
 core is up to date         0,0   B  0,00B/s 00:00 [----------------------]   0%
 extra is up to date        0,0   B  0,00B/s 00:00 [----------------------]   0%
 community is up to date    0,0   B  0,00B/s 00:00 [----------------------]   0%
 multilib is up to date     0,0   B  0,00B/s 00:00 [----------------------]   0%
:: Starting full system upgrade...
 there is nothing to do
```
```
$ yaourt -Syua
[sudo] password for xfce: 
:: Synchronising package databases...
 core is up to date         0,0   B  0,00B/s 00:00 [----------------------]   0%
 extra is up to date        0,0   B  0,00B/s 00:00 [----------------------]   0%
 community is up to date    0,0   B  0,00B/s 00:00 [----------------------]   0%
 multilib is up to date     0,0   B  0,00B/s 00:00 [----------------------]   0%
 Foreign packages: | 16 / 16
```

![pamac](https://user-images.githubusercontent.com/11800714/31880481-f4f2410a-b7e0-11e7-9056-8aaaac966fcc.png)

**Comment by guinux** [comment]
Ok do not change pkgrel so I can test and fix it.

**Comment by oberon2007** [comment]
Thank you! I won't touch it 😉 👍

**Comment by oberon2007** [comment]
On my end the behaviour has suddenly disappeared after some package updates. Not sure why... Just had a few package updates but nothing that would seem connected.

**Comment by oberon2007** [comment]
Maybe the reason is that I had the two packages installed in the chroot locally, before they were added to the repo. I now just built another package and the chroot db got updated. Now the issue has disappeared.
In any case I don't think it's urgent to change anything. 😉

**Comment by guinux** [comment]
Yes it's certainly the fact you built the packages on your system because I just realized that sndio was installed this morning from repos on my system when I upgraded vlc-nightly and pamac doesn't show any updates.

**Comment by oberon2007** [comment]
Yes. Funny thing just was that removing the packages from the system and removing the actual packages from the cache before re-installing was not enough. Only when the chroot database got updated, pamac stopped complaining 😉

**Comment by oberon2007** [comment]
Now pamac is still acting up on me. Can't really figure out why:

![aur_pkgs](https://user-images.githubusercontent.com/11800714/32112452-9d66aa74-bb3d-11e7-9906-e39b40f6bffe.png)

**Comment by oberon2007** [comment]
This time it seems the update mesasge was triggered by a (legitimate) AUR update for **skypeforlinux-bin**.

**Comment by philmmanjaro** [comment]
what happens if you search for **vlc-nightly**? on my end it says it doesn't exist in the AUR, even we know it does :smile: 

Looks fine on my end. Only AUR packages and no conflicts with installed binaries having the same name ...

![bildschirmfoto_2017-10-27_19-08-05](https://user-images.githubusercontent.com/5050022/32116270-3db3313a-bb4a-11e7-9d69-98655695b4e8.png)

**Comment by oberon2007** [comment]
True! pamac thinks vlc-nightly only exists in the repo and not in AUR, while in reality it exists in both.

**Comment by oberon2007** [comment]
And it's the same for all the packages it wants to update for me. mugshot, xcurosr-breeze, ...

**Comment by guinux** [comment]
Pamac doesn't show a package from AUR if it's also available in a repo.
Pamac checks for an update from AUR only if the package is not provided by a repo.
It seems related to your local repo, does pamac find mugshor (for example) in the repos ?

**Comment by oberon2007** [comment]
Yes, pamac lists all these packages as available from repos. At the same time the update page shows the higher pkgrels in AUR:
![updates](https://user-images.githubusercontent.com/11800714/32121583-7f3e4cd0-bb5d-11e7-8def-ee66b6b83fc5.png)

**Comment by oberon2007** [comment]
pkg and iso chroots are up-to-date. none of these packages have been locally installed or installed to the chroot locally.

**Comment by philmmanjaro** [comment]
And when you try to update it uses the AUR packages?

**Comment by oberon2007** [comment]
No. It doesn't update anything. I get stuck in a loop, where 'Apply' remains the only available button:

![update](https://user-images.githubusercontent.com/11800714/32122144-a853ba86-bb5f-11e7-957f-8119c0985def.png)

**Comment by guinux** [comment]
I have to say I can't find why you experienced this trouble.

**Comment by petsam** [comment]
Could that behavior have been caused because of mirror differences or the sequence of checking them in pamac?

**Comment by andruska** [comment]
I can report the same issue. Pamac offers updates over community packages. Particular packages are palemoon, mugshot and xcursor-breeze.

**Comment by andruska** [comment]
I can report the same issue. Pamac offers AUR updates over packages in community repo. Particular packages are palemoon, mugshot and xcursor-breeze.
